easybuild/easyblocks/p/petsc.py

```diff
@@ -262,7 +262,9 @@ def configure_step(self):
                         ss_libs = [x if x != "SLIP_LU" else "SPEX" for x in ss_libs]
 
                     suitesparse_inc = os.path.join(suitesparse, "include")
-                    inc_spec = "-include=[%s]" % suitesparse_inc
+                    # SuiteSparse can install its headers into a subdirectory of the include directory instead.
+                    suitesparse_incs = [suitesparse_inc, os.path.join(suitesparse_inc, 'suitesparse')]
+                    inc_spec = "-include=[%s]" % ','.join(suitesparse_incs)
```

Comment:
```suggestion
                    suitesparse_incs = [suitesparse_inc]
                    # SuiteSparse can install its headers into a subdirectory of the include directory instead.
                    suitesparse_inc_subdir = os.path.join(suitesparse_inc, 'suitesparse')
                    if os.path.exists(suitesparse_inc_subdir):
                      suitesparse_incs.append(suitesparse_inc_subdir)
                    inc_spec = "-include=[%s]" % ','.join(suitesparse_incs)
```
Maybe something like this?